### PR TITLE
oadp-1.2: OADP-2377 Add mustgather registry image to relatedImage

### PR DESCRIPTION
**Pull request by kaovilai**
cp https://github.com/openshift/oadp-operator/pull/1135

### CODE REVIEW

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -809,6 +809,8 @@ spec:
                   value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
                 - name: RELATED_IMAGE_DATA_MOVER_DUMMY_POD
                   value: quay.io/konveyor/rsync-transfer:latest
+                - name: RELATED_IMAGE_MUSTGATHER_RHEL8
```
Comment (rayfordj) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:812]:
```suggestion
                - name: RELATED_IMAGE_MUSTGATHER
```
must it include an EL designation?

File: config/manager/manager.yaml [hunk 2]
```diff
@@ -58,6 +58,8 @@ spec:
             value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
           - name: RELATED_IMAGE_DATA_MOVER_DUMMY_POD
             value: quay.io/konveyor/rsync-transfer:latest
+          - name: RELATED_IMAGE_MUSTGATHER_RHEL8
```
Comment (rayfordj) [inline config/manager/manager.yaml:61]:
```suggestion
          - name: RELATED_IMAGE_MUSTGATHER
```

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -809,6 +809,8 @@ spec:
...
+                  value: registry.redhat.io/oadp/oadp-mustgather-rhel8
```
Comment (rayfordj) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:813]:
```suggestion
                  value: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
```

Should use a complete pullspec here...

```
$ podman pull registry.redhat.io/oadp/oadp-mustgather-rhel8
Trying to pull registry.redhat.io/oadp/oadp-mustgather-rhel8:latest...
WARN[0000] Failed, retrying in 1s ... (1/3). Error: initializing source
docker://registry.redhat.io/oadp/oadp-mustgather-rhel8:latest: reading
manifest latest in registry.redhat.io/oadp/oadp-mustgather-rhel8: unsupported:
This repository does not use the "latest" tag to track the most recent image
and must be pulled with an explicit version or image reference. For more
information, see: https://access.redhat.com/articles/4301322
```
vs
```
$ podman pull registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2 
Trying to pull registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2...
Getting image source signatures
Checking if image destination supports signatures
Copying blob 7894e63714fa done  
Copying blob 5329d7039f25 skipped: already exists  
Copying config 20ec50564a done  
Writing manifest to image destination
Storing signatures
20ec50564a8e75366350aa2424b67344c12226cf922eea1341d9b335bfd1d92d
```

**Review by rayfordj — CHANGES_REQUESTED** [review]

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -937,4 +939,6 @@ spec:
     name: kubevirt-velero-plugin
   - image: quay.io/konveyor/rsync-transfer:latest
     name: data-mover-dummy-pod
+  - image: registry.redhat.io/oadp/oadp-mustgather-rhel8
+    name: mustgather-rhel8
```
Comment (rayfordj) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:943]:
```suggestion
    name: mustgather
```

**Review by rayfordj — APPROVED** [review]
/lgtm

**Review by sseago — APPROVED** [review]

**Pull request merged**